test/routing/Page.test.tsx

```diff
@@ -72,7 +72,7 @@ describe.skip('Routing - Page', () => {
     );
   });
 
-  test('switches page content to go back a page', async () => {
+  test.skip('switches page content to go back a page', async () => {
```

Comment:
I haven't been able to get this one to pass yet because there are no `<va-button>`s or button pairs rendered on the page. If I put a debug call right before the querySelector for `button.prev`, we can see that there's no button to click on:

```diff
diff --git a/test/routing/Page.test.tsx b/test/routing/Page.test.tsx
index 09fb4f2..e5f5e7a 100644
--- a/test/routing/Page.test.tsx
+++ b/test/routing/Page.test.tsx
@@ -1,5 +1,6 @@
 import React from 'react';
 import { render, waitFor } from '@testing-library/react';
+import { screen } from '@testing-library/dom'
 
 import { MemoryRouter, Route, Routes } from 'react-router-dom-v5-compat';
 import { FormRouterProps } from '../../src/routing/types';
@@ -72,7 +73,7 @@ describe('Routing - Page', () => {
     );
   });
 
-  test.skip('switches page content to go back a page', async () => {
+  test('switches page content to go back a page', async () => {
     const { container } = render(
       <MemoryRouter initialEntries={['/', '/page-two']} initialIndex={1}>
         <FormRouterInternal
@@ -85,6 +86,7 @@ describe('Routing - Page', () => {
         </FormRouterInternal>
       </MemoryRouter>
     );
+    screen.debug();
     act(() => {
       const goLink = container.querySelector('button.prev');
       goLink?.dispatchEvent(new MouseEvent('click', { bubbles: true }));
```

The output of the debug call:

![Screenshot of the test output with a failing test. The test fails on the expectation for the inner HTML of an h3 to contain "page one", and the logged markup shows that it contains "page two" there are no buttons to be clicked.](https://user-images.githubusercontent.com/2008881/182914507-a5a285ad-2be0-4a5f-b6aa-2bde237cbe2a.png)

Reply:
@bjmccotter7192 I think [these conditional rendering changes](https://github.com/department-of-veterans-affairs/va-forms-system-core/pull/508/files#diff-4d081acddafe6e0e9504f0c02a7d4d696922f397770355924663ccda937fafa3R61-R79) have caused this test to fail.

My understanding from that diff is that originally, the "Previous" button would be rendered if there was a previous route and the `hidePreviousButton` prop was falsey. Now, the previous button is rendered as part of a pair _if there is a next route_ as well as a previous route.

I'm seeing null for `nextRoute` when I run the test with these logs:

```diff
diff --git a/src/routing/Page.tsx b/src/routing/Page.tsx
index f21fbba..66cb6e3 100644
--- a/src/routing/Page.tsx
+++ b/src/routing/Page.tsx
@@ -27,6 +27,8 @@ export default function Page(props: PageProps): JSX.Element {
   const currentLocation = useLocation();
 
   const { nextRoute, previousRoute } = useContext(RouterContext);
+  console.log('next', nextRoute);
+  console.log('prev', previousRoute);
 
   useEffect(() => {
     // Reset the form on route change.
```

I don't know if the test should be updated or if the code should.

Reply:
I'm going to pull your branch and mess around with this because I could have ruined these tests with my previous push. I think the code needs to be updated to use the same logic so that those routes and buttons show up properly.